src/send-email.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const nodemailer = require('nodemailer');
+
+// create reusable transporter object using the default SMTP transport
+let transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: 'welcome.to.cahms@gmail.com',
+    pass: 'Cahms100'
+  }
+});
+
+// send mail with defined transport object
+function sendMail(mailOptions){
+  transporter.sendMail(mailOptions, (error, info) => {
```

Comment:
This function is asynchronous, `sendMail` should take a callback, and control should be returned to your handler.

You need to think about errors..